src/plz/plz.go

```diff
@@ -146,6 +147,7 @@ func findOriginalTask(state *core.BuildState, target core.BuildLabel, addToList
 		prefix := ""
 		if target.Subrepo != "" {
 			state.WaitForInitialTargetAndEnsureDownload(target.SubrepoLabel(state, ""), target)
+			time.Sleep(time.Millisecond * 100)
```

Comment:
Why do we need this? Is it that `WaitForInitialTargetAndEnsureDownload` doesn't block on the subrepo being initialised I guess?

Reply:
Ooops this shouldn't be here. This is because there's another race condition that I will fix in another PR.